app/components/message/message.css

```diff
@@ -11,3 +11,19 @@
   color: var(--color-text-light);
   font-size: var(--font-size-s);
 }
+
+.Message-photos {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  padding-bottom: var(--spacing-unit);
+}
+
+.Message-photos > img {
+  border: 3px solid rgba(0, 0, 0, 0.05);
+  max-width: 100%;
+  height: auto;
```

Comment:
`height: auto` should be the initial value of any `img` element, so I think this might not be necessary.

```suggestion
```